Week04/lesson08/homework/temp_converter_2/js/index.js

```diff
@@ -0,0 +1,36 @@
+function convertCToF() {
```

Comment:
So I think your biggest issue with this other version is a conceptional one more than an execution one. Basically, I think you got all tangled up by the fact that there's only ONE "convert" button but TWO possible things that could happen (convert C to F or convert F to C).

Back to basics: like in example one, you have ONE button that's going to trigger ONE function as a result:

```
function convert() {
  //Do stuff here
} 
document.querySelector("#submit").onclick = convert;
```

... whether you're translating C to F or vice-versa is logic that goes WITHIN that single `convert()` function.

I'll cover this in class tomorrow, but consider this:

```
function convert() {
  var c = document.querySelector('#c').value;
  var f = document.querySelector('#f').value;
  var result, scale;

  if (f === "") {
    //If the value of f is an empty string, then we know the user put the text into the c <input> and is looking to convert INTO f
  } else {
    //Since the value of f is NOT an empty string, then we know the value of c must be, so then we know the user put the text into the f <input> and is looking to convert INTO c
  }
  document.querySelector('h3').innerHTML = "Result: " + result + "°" + scale;
}
```